### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by clenk**
Added support for Python 3.3, 3.4, and 3.5.

### CODE REVIEW

File: ramrod/results.py [hunk 1]
```diff
@@ -46,16 +44,17 @@ def __unicode__(self):
...
-        return unicode(self.document)
+        return text_type(self.document)
 
 
     def __str__(self):
         if not self.document:
             return ''
 
-        return str(self.document)
+        return binary_type(self.document)
```
Comment (gtback) [inline ramrod/results.py]: Shouldn't __str__ on Py3 return a `str` (Unicode type)? I know there's a recommended way to do this, but I don't remember what it is (maybe `python_2_unicode_compatible` like you're using below)?

File: setup.py [hunk 2]
```diff
@@ -22,11 +22,8 @@ def get_version():
 
 py_maj, py_minor = sys.version_info[:2]
 
-if py_maj != 2:
-    raise Exception('stix-ramrod required Python 2.6/2.7')
-
 if (py_maj, py_minor) < (2, 6):
-    raise Exception('stix-ramrod requires Python 2.6/2.7')
+    raise Exception('stix-ramrod requires at least Python 2.6')
```
Comment (gtback) [inline setup.py]: I would prefer we say "2.6, 2.7 or 3.3+"

**Comment by gtback** [comment]
Looks good to me. Just a couple questions/comments (above)

**Comment by clenk** [comment]
Good points; see latest commit. I did some checks to make sure __str__ and __unicode__ in UpdateResults returned the same types in Python 2 as they did before any of my changes.

**Comment by gtback** [comment]
Thanks, @clenk ! I tweaked the version check a bit to ensure 3.0, 3.1, and 3.2 don't work.

**Pull request merged**